### ISSUE TITLE
the application scope  of the data and model 

### ISSUE THREAD

**Issue by aqsc** [facebookresearch/InterHand2.6M#39]
Thanks for this great work and dataset. Do  the dataset and model  just fit in the scenario that two hands are very close to each other?

**Comment by mks0601** [comment]
Good point. That is a kind of drawback of the proposed baseline (InterNet).
